### PR TITLE
rename ufrag to usernameFragment

### LINKED ISSUE #1210: ufrag vs usernameFragment
http://w3c.github.io/webrtc-pc/#dom-rtcicecandidate-ufrag
uses ufrag.

http://w3c.github.io/webrtc-pc/#dom-rtciceparameters-usernamefragment
uses usernameFragment.

They're the same. Can we use just one name?

### PR DESCRIPTION
**Pull request by fippo**
renames the ICE ufrag to usernameFragment for consistency.
Fixes #1210

***
[Preview](https://s3.amazonaws.com/pr-preview/fippo/webrtc-pc/ufrag.html) | [Diff](https://s3.amazonaws.com/pr-preview/w3c/webrtc-pc/a12bf47...fippo:0060df4.html)

### CODE REVIEW

File: webrtc.html [hunk 1]
```diff
@@ -4106,7 +4106,7 @@
               the candidate that it is derived from. For host candidates, the
               <code>relatedPort</code> is <code>null</code>. This corresponds to
               the <code>rel-port</code> field in <a><code>candidate-attribute</code></a>.</dd>
-              <dt><code><dfn>ufrag</dfn></code> of type <span class=
+              <dt><code><dfn>usernameFragment</dfn></code> of type <span class=
               "idlAttrType"><a>DOMString</a></span>, readonly, nullable</dt>
               <dd>This carries the <code>ufrag</code> as defined in section
```
Comment (fippo) [inline webrtc.html]: here ufrag is the right thing and doesn't need to be replaced.
But technically https://tools.ietf.org/html/rfc5245#section-15.4 talks about ice-ufrag, not ufrag...

Comment (taylor-b) [inline webrtc.html]: "ice-ufrag" is the attribute, "ufrag" is the field inside that attribute. So I think we're ok.

File: webrtc.html [hunk 2]
```diff
@@ -2703,9 +2703,9 @@
                         </ol>
                       </li>
                       <li>
-                        <p>If <code><var>candidate</var>.ufrag</code> is neither
+                        <p>If <code><var>candidate</var>.usernameFragment</code> is neither
                         <code>undefined</code> nor <code>null</code>, and is not
-                        equal to any ufrag present in the corresponding
+                        equal to any usernameFragment present in the corresponding
```
Comment (taylor-b) [inline webrtc.html]: Since this isn't code, should just say "username fragment" here like ICE itself does.

File: webrtc.html [hunk 3]
```diff
@@ -2716,8 +2716,8 @@
                         <p>In parallel, add the ICE candidate
                         <var>candidate</var> as described in <span data-jsep=
                         "addicecandidate">[[!JSEP]]</span>. Use
-                        <code><var>candidate</var>.ufrag</code> to identify the
-                        ICE <a>generation</a>; if the ufrag is null, process the
+                        <code><var>candidate</var>.usernameFragment</code> to identify the
+                        ICE <a>generation</a>; if the usernameFragment is null, process the
```
Comment (taylor-b) [inline webrtc.html]: Here we're talking about the member, not the username fragment in the abstract sense, so should be `<code>usernameFragment</code>`.

File: webrtc.html [hunk 4]
```diff
@@ -7687,7 +7687,7 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
           <code><a data-for="RTCIceCandidate">sdpMLineIndex</a></code>
           set to the values associated with this
           <code><a>RTCIceTransport</a></code>, with
-          <code><a data-for="RTCIceCandidate">ufrag</a></code> set to the ufrag
+          <code><a data-for="RTCIceCandidate">usernameFragment</a></code> set to the usernameFragment
```
Comment (taylor-b) [inline webrtc.html]: Here's another place.

**Comment by taylor-b** [comment]
I went ahead and addressed my comments in the online editor FYI.

**Comment by stefhak** [comment]
LGTM.

File: webrtc.html (hunk 1 above)
Comment (aboba) [inline webrtc.html]: Is line 2125 intentional or is that a typo?

**Pull request merged**